platform/core/src/classes/MetadataProvider.js

```diff
@@ -408,14 +415,18 @@ class MetadataProvider {
     //   SOPInstanceUID,
     // })
     // somewhere else
-    if (imageId.startsWith('wadors:')) {
+    if (
+      imageId.startsWith('wadors:') ||
+      imageId.startsWith('streaming-wadors:')
```

Comment:
The base metadata was registered specifically, but the remaining ones were missing, so it was throwing an exception.  This allows parsing the streaming-wadors URL the same way the base URL is parsed.